magellan-sample-advanced/src/main/java/com/wealthfront/magellan/sample/advanced/TideLocationsScreen.java

```diff
@@ -0,0 +1,62 @@
+package com.wealthfront.magellan.sample.advanced;
+
+import android.content.Context;
+import android.support.annotation.DrawableRes;
+import android.widget.Toast;
+
+import com.wealthfront.magellan.Screen;
+
+class TideLocationsScreen extends Screen<TideLocationsView> {
+
+    public enum TideLocations {
+        SAN_FRANCISCO(9414290, R.drawable.san_francisco, "San Francisco"),
+        SEATTLE(9447130, R.drawable.seattle, "Seattle"),
+        ANCHORAGE(9455920, R.drawable.anchorage, "Anchorage"),
+        GALVESTON(8771450, R.drawable.galveston, "Galveston"),
+        MIAMI(8723214, R.drawable.miami, "Miami"),
+        CHARLESTON(8665530, R.drawable.charleston, "Charleston"),
+        CHESAPEAKE_BAY(8638863, R.drawable.chesapeake, "Chesapeake Bay"),
+        NEW_YORK(8518750, R.drawable.new_york, "New York"),
+        BOSTON(8443970, R.drawable.boston, "Boston"),
+        WASHINGTON_DC(8594900, R.drawable.washington_dc, "Washington D.C.");
```

Comment:
Where did you find the images? We need images that are free to use (one potential solution is to find them on wikipedia, or a website where you can search for free  to use images. In wikipedia they are usually Creative Common license.
We don't need it to be free to use commercially, since this is an Open Source project.